Dalamud/Game/Text/SeStringHandling/Payloads/AutoTranslatePayload.cs

```diff
@@ -21,6 +21,18 @@ public class AutoTranslatePayload : Payload, ITextProvider
     [JsonProperty]
     private uint key;
 
+    /// <summary>
+    /// Gets the sheet group this entry belongs to
+    /// </summary>
+    [JsonIgnore]
+    public uint Group => this.group;
+
+    /// <summary>
+    /// Gets the sheet key for this entry
+    /// </summary>
+    [JsonIgnore]
+    public uint Key => this.key;
```

Comment:
Why not just replace the fields with these properties and use the relevant autos? The fields are `private` already so that shouldn't be a breaking change.